src/composables/use-i18n-sync.js

```diff
@@ -130,7 +129,6 @@ export default function useI18nSync() {
   onMounted(() => {
     window.addEventListener('message', localeMsgHandler)
     sendWindowMessage({
```

Comment:
Can we import `dev` from the `utils/dev` here, too? It's usually helpful to see the message exchange when debugging the iframe.